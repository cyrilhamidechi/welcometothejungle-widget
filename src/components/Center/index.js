import PropTypes from "prop-types";
import React from "react";

import * as S from "./styled";

const Center = ({ children }) => <S.Wrapper>{children}</S.Wrapper>;

Center.propTypes = {
  children: PropTypes.node.isRequired
};

export default Center;
