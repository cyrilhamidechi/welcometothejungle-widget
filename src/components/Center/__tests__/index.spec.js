import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Center from "../";

it("should render the component correctly", () => {
  const tree = renderer.create(<Center>test</Center>).toJSON();

  expect(tree).toMatchSnapshot();
});
