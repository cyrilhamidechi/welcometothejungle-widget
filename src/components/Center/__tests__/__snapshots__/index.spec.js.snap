// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
>
  test
</div>
`;
